### PR TITLE
fix named exports for es6 bundles, update rollup deps

### PR DESCRIPTION
**Pull request by ezekielchentnik**

### CODE REVIEW

**Comment by developit** [comment]
Hey @ezekielchentnik - was just about to copy the rollup config from `preact-compat` and noticed this PR.  I'm tempted to try out `preact-compat`'s since it's just the one file (keeps things clean) - if I can't get that working this looks like the ticket.

File: rollup.config.js [hunk 1]
```diff
@@ -61,7 +42,6 @@ export default {
 		nodeResolve({
 			jsnext: true,
 			module: true,
-			skip: ['react', 'preact'],
```
Comment (developit) [inline rollup.config.js]: I think dropping these bundles preact into the package

**Comment by ezekielchentnik** [comment]
@developit sounds good, definitely cleaner

File: rollup.config.js (hunk 1 above)
Comment (ezekielchentnik) [inline rollup.config.js]: I think `skip` is deprecated with latest version of plugin and `external` is preffered

Comment (developit) [inline rollup.config.js]: ah that makes sense, the two were sortof redundant

**Comment by ezekielchentnik** [comment]
also, I believe these rollup updates will fix a couple outstanding import issues, #14, #1 etc.  multiple birds 1 stone :)

**Comment by developit** [comment]
totally - was pumped when I saw it's already done haha

**Comment by developit** [comment]
oooh - is this what I think it is? you're making my job too easy man haha

**Comment by ezekielchentnik** [comment]
it is, i've condensed my original rollup tweaks into 1 config (like preact-compat); although, I'm noticing bundle size increase :( - troubleshooting that

**Comment by ezekielchentnik** [comment]
uglify is not dropping as much with this change

**Comment by developit** [comment]
Hmm - might also be an upstream change in react-redux causing the size increase?

**Comment by developit** [comment]
@ezekielchentnik good call on proptypes, though we might need to do 2 modules there (to not trample the default exports for react imports)

**Comment by ezekielchentnik** [comment]
@developit what's needed to get this merged?

**Comment by developit** [comment]
Nothing! My GH inbox is doomed

**Pull request merged**